include/openssl/target.h

```diff
@@ -82,6 +82,17 @@
 #define OPENSSL_APPLE
 #endif
 
+#if defined(__APPLE__)
```

Comment:
NP: There's another `#if defined(__APPLE__)` 4 lines above this.  These two could be merged.